### ISSUE TITLE
How do you disable the interface generation?

### ISSUE THREAD

**Issue by dodyg** [mganss/XmlSchemaClassGenerator#48]
I tried

``` dotnet XmlSchemaClassGenerator.Console.dll *.xsd -o output -n http://www.iata.org/IATA/EDIST/2017.2=IATA -n==XmlSchema -v -ct System.Collections.Generic.List`1 -t false```

``` dotnet XmlSchemaClassGenerator.Console.dll *.xsd -o output -n http://www.iata.org/IATA/EDIST/2017.2=IATA -n==XmlSchema -v -ct System.Collections.Generic.List`1 -t false```

``` dotnet XmlSchemaClassGenerator.Console.dll *.xsd -o output -n http://www.iata.org/IATA/EDIST/2017.2=IATA -n==XmlSchema -v -ct System.Collections.Generic.List`1 -t 0```

At this moment I am getting compilation error related to interface such as 

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0738	'ServiceListRS' does not implement interface member 'IIATA_PayloadStdAttributes.TransactionStatusCode'. 'ServiceListRS.TransactionStatusCode' cannot implement 'IIATA_PayloadStdAttributes.TransactionStatusCode' because it does not have the matching return type of 'IIATA_PayloadStdAttributesTransactionStatusCode'.	NDC.Xml	E:\NDC\NDC.Xml\IATA.cs	115130	Active
Error	CS0738	'ServiceListRS' does not implement interface member 'IIATA_PayloadStdAttributes.Target'. 'ServiceListRS.Target' cannot implement 'IIATA_PayloadStdAttributes.Target' because it does not have the matching return type of 'IIATA_PayloadStdAttributesTarget'.	NDC.Xml	E:\NDC\NDC.Xml\IATA.cs	115130	Active
```

My thinking is that without interface generation, these errors would go away.

**Comment by mganss** [comment]
Just like it says in the readme: "Append - to option to disable it, e.g. --interface-" (you can also use the short form -t-).

I've released 2.0.83-beta which fixes the interface problem.

**Comment by dodyg** [comment]
This is amazing.
